lib/msf/core/post/windows/shadowcopy.rb

```diff
@@ -182,6 +182,25 @@ def start_vss
         return false
       end
     end
+    unless start_swprv
+      return false
+    end
+    return true
+  end
+
+  def start_swprv
+    vss_state = wmic_query('Service where(name="swprv") get state')
```

Comment:
This argument can be moved to a separate Github issue, if you feel that strongly about it. It ultimately has nothing to do with the functionality addressed by this PR, and would be a change to an existing library, so it shouldn't be polluted in this pr